src/programs/project3d/project3d.cpp

```diff
@@ -60,8 +77,6 @@ void Project3DApp::DoInteractiveUserInput()
 	wanted_SNR = my_input->GetFloatFromUser("Wanted SNR", "The ratio of signal to noise variance after adding Gaussian noise and before masking", "1.0", 0.0);
 	padding = my_input->GetFloatFromUser("Padding factor", "Factor determining how much the input volume is padded to improve projections", "1.0", 1.0);
 	my_symmetry = my_input->GetSymmetryFromUser("Particle symmetry", "The assumed symmetry of the particle to be reconstructed", "C1");
-	apply_CTF = my_input->GetYesNoFromUser("Apply CTF", "Should the CTF be applied to the output projections?", "No");
-	apply_shifts = my_input->GetYesNoFromUser("Apply shifts", "Should the particle translations be applied to the output projections?", "No");
 	apply_mask = my_input->GetYesNoFromUser("Apply mask", "Should the particles be masked with the circular mask?", "No");
 	add_noise = my_input->GetYesNoFromUser("Add noise", "Should the Gaussian noise be added?", "No");
```

Comment:
It's a bit strange to ask for the SNR and then a few questions later ask whether to add noise at all. It'd be nicer to aks whether to add noise, and if user says yes, ask for SNR...

Reply:
Having a variable number of inputs is annoying for scripting. Couldn't the same thing be accomplished by combining them into one? Get rid of add_noise, and if wanted_snr = 0, don't add noise.

Probably change the default from 1.0 --> 0.0 then

wanted_SNR = my_input->GetFloatFromUser("Wanted SNR", "The ratio of signal to noise variance after adding Gaussian noise and before masking. To leave unmodified, adding no noise, set to 0.0", "0.0", 0.0);

Reply:
Yes,  this was the way it was originally written - I'll leave it for a future change as the goal here was to add functionality rather than clean it up